verl/trainer/main_ppo.py

```diff
@@ -155,14 +156,17 @@ def run(self, config):
 
         resource_pool_manager = ResourcePoolManager(resource_pool_spec=resource_pool_spec, mapping=mapping)
 
+        device_name = "npu" if is_npu_available else "cuda"
```

Comment:
We found that device_name is set to `cuda`, since is_npu_available is False because TaskRunner is a cpu actor with `ASCEND_RT_VISIBLE_DEVICES=""`, torch.npu.is_available() return False.